src/main/resources/application.yml

```diff
@@ -20,4 +21,6 @@ mpesa:
     client-key: 0pLxbN83FrOl5Nd0Fh9Zi5BQlMxSL2n5
     client-secret: YzuGNoJxeub8ZC6d
   api:
-    lipana: https://sandbox.safaricom.co.ke/mpesa/stkpush/v1/processrequest
\ No newline at end of file
+    lipana: https://sandbox.safaricom.co.ke/mpesa/stkpush/v1/processrequest
+    transaction-status: https://sandbox.safaricom.co.ke/mpesa/transactionstatus/v1/query
```

Comment:
Break these further. Put host as https://sandbox.safaricom.co.ke and liana path as /mpesa/stkpush/v1/processrequest.

Reply:
done